index.ts

```diff
@@ -241,9 +244,7 @@ const loadOmnivore = async (
     if (targetBlock) {
       await logseq.Editor.updateBlock(
         targetBlock.uuid,
-        `${blockTitle} [:small.opacity-20 "fetched at ${new Date(
-          lastSavedAt
-        ).toISOString()}"]`
+        `${blockTitle} [:small.opacity-20 "fetched at ${new Date().toISOString()}"]`
```

Comment:
This is the fix i was talking about where it was using the same sync datte over and over again and never changing

Reply:
Nice one!